src/borg/testsuite/archiver.py

```diff
@@ -1044,6 +1044,25 @@ def test_create_stdin(self):
         extracted_data = self.cmd('extract', '--stdout', self.repository_location + '::test', binary_output=True)
         assert extracted_data == input_data
 
+    def test_create_content_from_command(self):
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+        input_data = 'some test content'
+        name = 'a/b/c'
+        self.cmd('create', '--stdin-name', name, '--content-from-command',
+                 self.repository_location + '::test', 'echo', input_data)
```

Comment:
no '--' here separating borg and echo command?

Reply:
even if it is not strictly needed, i guess we better test if it also works with `--`.

Reply:
agreed